noel/setrun.py

```diff
@@ -0,0 +1,484 @@
+    #  [minlevel,maxlevel,t1,t2,x1,x2,y1,y2]
+    # Noel guages from NOAA stations and newspapers
+    rundata.gaugedata.gauges.append([1, -80.16 , 25.73,
+                                     rundata.clawdata.t0,
+                                     rundata.clawdata.tfinal])
+    rundata.gaugedata.gauges.append([2, -81.81, 24.54,
+                                     rundata.clawdata.t0,
+                                     rundata.clawdata.tfinal])
+    rundata.gaugedata.gauges.append([3, -67.94, 18.09,
+                                     rundata.clawdata.t0,
+                                     rundata.clawdata.tfinal])
+    rundata.gaugedata.gauges.append([4, -67.05, 17.97,
+                                     rundata.clawdata.t0,
+                                     rundata.clawdata.tfinal])
+    rundata.gaugedata.gauges.append([5, -74.90, 23.65,
+                                     rundata.clawdata.t0,
+                                     rundata.clawdata.tfinal])
+    rundata.gaugedata.gauges.append([6, -75.44, 24.29,
+                                     rundata.clawdata.t0,
+                                     rundata.clawdata.tfinal])
+
+    # Force the gauges to also record the wind and pressure fields
+    rundata.gaugedata.aux_out_fields = [4, 5, 6]
+
+    # ------------------------------------------------------------------
+    # GeoClaw specific parameters:
+    # ------------------------------------------------------------------
+    rundata = setgeo(rundata)
+
+    return rundata
+    # end of function setrun
+    # ----------------------
+
+
+# -------------------
+def setgeo(rundata):
+    """
+    Set GeoClaw specific runtime parameters.
+    For documentation see ....
+    """
+
+    geo_data = rundata.geo_data
+
+    # == Physics ==
+    geo_data.gravity = 9.81
+    geo_data.coordinate_system = 2
+    geo_data.earth_radius = 6367.5e3
+    geo_data.rho = 1025.0
+    geo_data.rho_air = 1.15
+    geo_data.ambient_pressure = 101.3e3
+
+    # == Forcing Options
+    geo_data.coriolis_forcing = True
+    geo_data.friction_forcing = True
+    geo_data.friction_depth = 1e10
+
+    # == Algorithm and Initial Conditions ==
+    geo_data.sea_level = 0.0
+    geo_data.dry_tolerance = 1.e-2
+
+    # Refinement Criteria
+    refine_data = rundata.refinement_data
+    refine_data.wave_tolerance = 1.0
+    refine_data.speed_tolerance = [1.0, 2.0, 3.0, 4.0]
+    refine_data.deep_depth = 300.0
+    refine_data.max_level_deep = 4
+    refine_data.variable_dt_refinement_ratios = True
+
+    # == settopo.data values ==
+    topo_data = rundata.topo_data
+    topo_data.topofiles = []
+    # for topography, append lines of the form
+    #   [topotype, minlevel, maxlevel, t1, t2, fname]
+    # See regions for control over these regions, need better bathy data for
+    # the smaller domains
+
+    # use topography data from seperate noel directory
+    topo_path_nc = os.path.join(datadir, 'noel_topo.nc')
```

Comment:
We need to remove the translation of the NetCDF file from the `setrun` file and just refer to the `tt3` file.  Instead we should describe the process of creating the `tt3` file.

Reply:
Hi should I do the translation in another file then?